sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3184,8 +3184,14 @@ SIREPO.app.directive('sbatchOptions', function(appState) {
                 if  (sbatchLoginStatusService.loggedIn === undefined) {
                     return null;
                 }
-                var s = 'conntected to ' + authState.jobRunModeMap[appState.models[$scope.simState.model].jobRunMode];
-                s = (sbatchLoginStatusService.loggedIn ? '' : 'not ') + s;
+                var s = 'connected to ' +
+                    authState.jobRunModeMap[appState.models[$scope.simState.model].jobRunMode];
+                if (sbatchLoginStatusService.loggedIn) {
+                    s += '. To start press "' + $scope.simState.startButtonLabel() + '"';
```

Comment:
@robnagler this will appear even when the user has frames and may not want to start a new simulation. Is that a problem?

Also, the text is long. I couldn't come up with anything shorter but open to suggestions.

Reply:
@bruhwiler what say you?